### ISSUE TITLE
v16.3.2 can be incompatible with EF Core 3.1 projects

### ISSUE THREAD

**Issue by bhanson-techempower** [thepirat000/Audit.NET#355]
**Describe the bug**
We upgraded Audit.EntityFramework.Core from v16.2.1 to v16.3.2 and it seems to be incompatible with our build because we're not using EF 5.

```sh
error NU1605: Detected package downgrade: Microsoft.EntityFrameworkCore.Abstractions from 5.0.1 to 3.1.10. Reference the package directly from the project to select a different version. 
error NU1605:  Audit.NET.16.3.2.BugReport -> Audit.EntityFramework.Core 16.3.2 -> Microsoft.EntityFrameworkCore 5.0.1 -> Microsoft.EntityFrameworkCore.Abstractions (>= 5.0.1) 
error NU1605:  Audit.NET.16.3.2.BugReport -> Microsoft.EntityFrameworkCore.Abstractions (>= 3.1.10)
```

**To Reproduce**
https://github.com/bhanson-techempower/Audit.NET.16.3.2.BugReport

**Expected behavior**
ASP.NET/EF Core 3.1 is the latest LTS build. I would expect a major version bump from Audit.NET for the underlying bump to .NET 5.

**Libraries (specify the Audit.NET extensions being used including version):**
 - Audit.EntityFramework.Core: 16.2.1 or 16.3.2
 - Microsoft.EntityFrameworkCore.Abstractions: 3.1.10

**Target .NET framework:**
 - .NET Core 3.1

**Additional context**
For now, we'll stay on v16.2.1. 

Thank you!

**Comment by thepirat000** [comment]
After this issue #352 I had to choose between EF Core 3 and 5 since 5 is not backward compatible at the level needed by this audit library.



Maybe I should have changed the major version, but I see no way to provide compatibility with both, except by providing a new package that forces EF Core 3 for netstandard 2.1.

So currently your only option is to use version 16.2.1 or older

**Comment by thepirat000** [comment]
I've created teo new packages for this pupose, when you need to use version 3 of Entity Framework Core, but are targeting .NET Standard >= 2.0.

In that case you must reference the packages ending with `.v3`

`Audit.EntityFramework.Core.v3`
or
`Audit.EntityFramework.Identity.Core.v3`

Please update your references to use this new package(s) and re-test

**Comment by thepirat000** [comment]
I've created two new packages for this purpose; when you need to use version 3 of Entity Framework Core, but are targeting .NET Standard >= 2.0.

In that case you must reference the packages ending with `.v3`

`Audit.EntityFramework.Core.v3`
or
`Audit.EntityFramework.Identity.Core.v3`

Please update your references to use this new package (starting on version 16.4.0) and re-test

**Comment by bhanson-techempower** [comment]
We upgraded to `v16.4.0` of `Audit.EntityFramework.Core.v3` and it is working! Thank you!

**Comment by daveraine** [comment]
@thepirat000 I've just tried switching from `Audit.EntityFramework.Identity.Core` 16.2.1 to `Audit.EntityFramework.Identity.Core.v3` 16.4.2 but this package depends on `Audit.EntityFramework.Core` and not `Audit.EntityFramework.Core.v3` so it's still upgrading EF Core to 5 when I use this package. Could you make it depend on `Audit.EntityFramework.Core.v3` so I can make the switch?

**Comment by thepirat000** [comment]
oh you're right, I've fixed that on version 16.4.3, please upgrade your references and re-test

**Comment by daveraine** [comment]
Awesome, thanks for the quick turnaround. It is working as expected now.
